src/modules/ethereum/chains/helpers.ts

```diff
@@ -0,0 +1,9 @@
+import { normalizeChainId } from 'src/shared/normalizeChainId';
+
+export function getCustomNetworkId(chainId: string) {
+  return `zerion-extension-${normalizeChainId(chainId)}`;
```

Comment:
maybe something like `client-id-`? This way it could be theoretically reusable between clients :)

Reply:
This is valid. However, I would probably mention Zerion in the template to decrease probability of collisions with real network names